tools/sampcd_processor.py

```diff
@@ -248,6 +248,7 @@ def __init__(
         self.mode = mode
         self.verbose = verbose
         self.config = {**XDOCTEST_CONFIG, **(config or {})}
+        self._test_capacity = {}
```

Comment:
```suggestion
        self._test_capacity = set()
```

`test_capacity` 应该是个 `set`？`{}` 是空 dict

Reply:
疏忽了 ... ...